docs/policies/policies.md

```diff
@@ -0,0 +1,480 @@
+
+### Semantic Model
+
+The ODRL Information Model defines the underlying semantic model for permission, prohibition, and obligation statements describing content usage. The information model covers the core concepts, entities and relationships that provide the foundational model for content usage statements. These machine-readable policies may be linked directly with the content they are associated to with the aim to allow consumers to easily retrieve this information.
+
+#### Most Important Terminology
+
+| Term     | Description |
+| -------- | -------- |
+| Policy | A group of one or more Rules
+| Rule | An abstract concept that represents the common characteristics of Permissions, Prohibitions, and Duties
+| Asset | A rsource or a collection of resources that are the subject of a Rule
+| Action | An operation on an Asset
+| Permission | The ability to exercise an Action over an Asset
+| Prohibition | The inability to exercise an Action over an Asset
+| Constraint | A boolean/logical expression that refines an Action and Asset collection or the conditions applicable to a Rule.
+
+## Policies in Catena-X
+
+A Catena-X policy uses the ODRL Information Model as a semantic model. Theoretically, a policy can be constructed individually using this ODRL Information Model. Catena-X does not yet use all the possibilities that ODRL provides, so the next example will show how a Catena-X policy is basically structured.
+
+### Policy Template Example with @context: `"http://www.w3.org/ns/odrl.jsonld"`
+
+    {
+      "@context": {},
+      "@type": "PolicyDefinitionRequest",
+      "@id": "{{POLICY_ID}}",
+      "policy": {
+        "@type": "Set",
+        "@context": "http://www.w3.org/ns/odrl.jsonld",
+        "permission": [
+          {
+            "action": "use",
+            "constraint": [
+              {
+                "leftOperand": {
+                  "@value": "<field>"
+                },
+                "operator": "eq",
+                "rightOperand": "<value>"
+              }
+            ]
+          }
+        ]
+      }
+    }

```

Comment:
Please put this example into code fences

Reply:
done